owl/EASE-ACT.owl

```diff
@@ -837,8 +837,13 @@ Note that buying an object is NOT PhysicalGetting. Buying, or ownership transfer
         <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Prospecting"/>
         <rdfs:subClassOf>
             <owl:Restriction>
-                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasParticipant"/>
-                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Plan"/>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isTaskOf"/>
```

Comment:
Can you explain this definition? If I understand it correctly, it is saying that Planning is a subclassof isTaskOf some (classifies some Plan)

Reply:
*Planning* is a sub-class of *Task*. So there are objects playing a role when the task is executed (*isTaskOf*), and one of these roles classifies the *Plan* (basically the *Outcome* or so)

Reply:
So before the statement was "Planning has some participant which is a plan", and this I have changed to "Planning is a task of some role that is taken by the plan"